### ISSUE TITLE
TypeScript compile error: Cannot find module 'prismjs' or its corresponding type declarations

### ISSUE THREAD

**Issue by haines** [FormidableLabs/prism-react-renderer#195]
Because `@types/prismjs` is a dev dependency, it's not installed when installing `prism-react-renderer` as a dependency. However, the generated types implicitly depend on it. 

```
node_modules/prism-react-renderer/dist/index.d.ts:3:34 - error TS2307: Cannot find module 'prismjs' or its corresponding type declarations.

3 import { Token as Token$1 } from 'prismjs';
                                   ~~~~~~~~~

node_modules/prism-react-renderer/dist/index.d.ts:4:34 - error TS2307: Cannot find module 'prismjs' or its corresponding type declarations.

4 export { default as Prism } from 'prismjs';
```

This can be resolved by installing `@types/prismjs` as well as `prism-react-renderer`, but it'd be nice if this wasn't necessary (or, at least, if it was documented).

**Comment by carloskelly13** [comment]
@haines This should be fixed in 2.0.3.

**Comment by haines** [comment]
Thanks @carloskelly13!
